secator/hooks/api.py

```
@@ -182,6 +183,27 @@ def resolve_workspace(value):
+
+	Returns (created, workspace_id): if a workspace with this name already exists in
+	the org it is reused (created=False) instead of erroring, so `secator ws use`
+	is safe to re-run. Otherwise it is created (created=True).
+	"""
+	# Reuse an existing workspace of the same name rather than hitting the API's
+	# per-org name-uniqueness check. resolve_workspace raises if not found.
+	try:
+		existing_id, _ = resolve_workspace(name)
+		return False, existing_id
+	except Exception:
+		pass
```

Ruff (0.15.17)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)